package.json

```diff
@@ -22,21 +22,25 @@
   "scripts": {
     "ng": "ng",
     "start": "run-p mock:server 'ng serve'",
+    "start:ff4j-properties": "run-p mock:server 'ng serve ff4j-properties -- --port=4202'",
```

Comment:
Nice work. Do we need `--` twice ?

Reply:
yes Paul, it was not taking the specified port.
We can use placeholders to give the arguments preceded by -- to scripts.
$ run-p "start-server -- --port {1}" -- 8080

Reply:
ok